tests/simulation/test_start_simulator.py

```diff
@@ -27,19 +28,19 @@ class TestStartSimulator:
 
     @pytest_asyncio.fixture(scope="function")
     async def get_chia_simulator(
-        self, automated_testing: bool = False, chia_root: Optional[Path] = None, config: Optional[Dict[str, Any]] = None
-    ) -> AsyncGenerator[Tuple[FullNodeSimulator, Path, Dict[str, Any], str, int], None]:
-        async for simulator_args in get_full_chia_simulator(automated_testing, chia_root, config):
+        self, tmp_path: Path, empty_keyring: Keychain
+    ) -> AsyncGenerator[Tuple[FullNodeSimulator, Path, Dict[str, Any], str, int, Keychain], None]:
+        async for simulator_args in get_full_chia_simulator(tmp_path, empty_keyring):
             yield simulator_args
 
     @pytest.mark.asyncio
     async def test_start_simulator(
```

Comment:
WE need to pass args to get_chia_simulator using pytest.

Reply:
pytest wasn't passing any arguments at all previously.

Reply:
I'm not sure what you are referring to, or suggesting, here Jack.  Could you say a bit more about it?